sdk/src/feature_set.rs

```diff
@@ -49,6 +49,17 @@ pub mod full_inflation {
         }
     }
 
+    pub mod bl {
+        pub mod vote {
+            // The private key for this address is held by the Solana Foundation
+            solana_sdk::declare_id!("HRzoLj4jufnYEWosm9kWVgBVFdxAuqB1hu7vLckCuQHa");
+        }
+        pub mod enable {
+            // The private key for this address is held by BL
+            solana_sdk::declare_id!("BLxyQtJPzYZLHyj1p9n5QHUvbPoJt4TtRh7BXbG4M6rR.json");
```

Comment:
Hey @brianlong, I removed the ".json" suffix here. Can you please confirm that the what remains here is in fact your public key:
```suggestion
            solana_sdk::declare_id!("BLxyQtJPzYZLHyj1p9n5QHUvbPoJt4TtRh7BXbG4M6rR");
```

Reply:
Gads! Sorry I missed that! Yes, this looks correct. Thanks for double-checking.